app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.js

```diff
@@ -182,28 +182,6 @@ async function addEthereumChainHandler(
     return end();
   }
 
-  let endpointChainId;
```

Comment:
why is this section of code removed?

Reply:
> why is this section of code removed?

these checks were moved to the confirmation template